### ISSUE TITLE
Up for assigning - Documenting AL constants

### ISSUE THREAD

**Issue by YanniZ06** [YanniZ06/HaxeAL-Soft#1]
OpenAL has a lot of constants set to be used as parameters in various get and set functions, some of which even ambigiously work with multiple functions.

The constants are already in the source:

![image](https://github.com/YanniZ06/HaxeAL-Soft/assets/102467588/9fc8c1b6-6866-4a86-a253-847a1020f1dd)

It would be pretty nice to have them documented properly for the API webpage that will be set up once this is done (using dox).
I could very much do this myself, however im trying to focus more on the main important stuff for now and anyone that (for some reason) feels like doing this can leave a comment and I'll assign them this "issue".

For information on the constants you can and probably _should_ use the openal soft documentation page (https://www.openal.org/documentation/OpenAL_Programmers_Guide.pdf).
You may also freely directly copy the information given to a degree, as the programmers guide will be listed as a source I used for the project anyways.

**Comment by YanniZ06** [comment]
Doing this myself or just not at all, not in demand as of now
